wodles/aws/aws_s3.py

```diff
@@ -841,11 +841,15 @@ def get_alert_msg(self, aws_account_id, log_key, event, error_msg=""):
         return alert_msg
 
     def find_account_ids(self):
-        return [common_prefix['Prefix'].split('/')[-2] for common_prefix in
-                self.client.list_objects_v2(Bucket=self.bucket,
-                                            Prefix=self.get_base_prefix(),
-                                            Delimiter='/')['CommonPrefixes']
-                ]
+        try:
+            return [common_prefix['Prefix'].split('/')[-2] for common_prefix in
+                    self.client.list_objects_v2(Bucket=self.bucket,
+                                                Prefix=self.get_base_prefix(),
+                                                Delimiter='/')['CommonPrefixes']
+                    ]
+        except KeyError:
+            print("ERROR: Invalid type of bucket")
```

Comment:
The error message is better than before, but I think it can be still improved. We should try to answer the following question the user is going to ask: 'and which is the right type of bucket??'

Reply:
Done!